packages/xo-web/src/common/xo-line-chart/index.js

```diff
@@ -721,7 +726,10 @@ export const IowaitChart = injectIntl(
             nValues: length,
             endTimestamp,
             interval,
-            valueTransform: value => `${value.toPrecision(2)}%`,
+            valueTransform: value =>
+              humanFormat(value, {
```

Comment:
Why do you need a human format for a percentage?

Reply:
Because it can exceed 100.

Reply:
Probably unnecessary, why not simply `round(2)`?